CHANGELOG.md

```diff
@@ -1,3 +0,0 @@
-ver 0.1.0 (2019-03-23)
```

Comment:
I think the tag/release notes is a better place to track the important bits. I've never been a fan of the CHANGELOG method for tracking these things. I suggest leaving it out.

Reply:
Fine by me.